commit 66c96415ba00ecc76f7052ecc3daec3742c00d4b (HEAD -> master)
Author: Trent Mick <trentm@gmail.com>
Date:   2019-06-11T12:05:44-07:00 (4 months ago)
    
    joyent/node-triton-audit-logger#4 clarify docs on `reqBody` and `resBody`

diff --git a/lib/audit-logger.js b/lib/audit-logger.js
index 038efb6..04432ab 100644
--- a/lib/audit-logger.js
+++ b/lib/audit-logger.js
@@ -329,6 +329,9 @@ function normalizeRouteOpts(ro, defaults) {
  *          or the integer value) at which to log. Default is "info".
  *      @param {Object} opts.reqBody - Options for logging request bodies.
  *      @param {Object} opts.resBody - Options for logging response bodies.
+ *          Each of `reqBody` and `resBody` are optional. If they are not
+ *          specified then there is no body logging. To enable body logging with
+ *          the defaults, specify an empty object: `reqBody: {}, reqBody: {},`.
  *          Each of `reqBody` and `resBody` may have the following fields.
  *          All fields are optional.
  *          - {Boolean} `include` - Whether to log the body. Default is true.
diff --git a/package.json b/package.json
index 63b5be0..8453dc1 100644
--- a/package.json
+++ b/package.json
@@ -36,6 +36,6 @@
     "restify": "^7.0.0",
     "restify-clients": "^2.6.4",
     "restify-errors": "^5.0.0",
-    "tap": "^12.5.2"
+    "tap": "^11.1.5"
   }
 }
